app/assets/javascripts/map.js

```diff
@@ -80,39 +100,101 @@ class Map {
       componentRestrictions: {country: 'ca'}
     };
 
-    const input = document.getElementById('address');
+    const input = document.getElementById('addressInput');
     const autocomplete = new google.maps.places.Autocomplete(input, options);
+    // TODO execute search when autocomplete option has been selected doesnt work
+    // autocomplete.addListener('place_changed', this.handleSearchSubmit);
   }
 
   addListeners() {
-    document.getElementById('AddressSearchModal').addEventListener('submit', this.handleSearchSubmit);
+    document.getElementById('addressSearchBar').addEventListener('keyup', this.enableFindButton)
+    document.getElementById('addressSearchBar').addEventListener('submit', this.handleSearchSubmit);
     this.drawingManager.addListener('polygoncomplete', this.handlePolygonCreated);
     this.removeLastControl.addEventListener('click', this.handleRemoveLastPolygon);
     this.removeAllControl.addEventListener('click', this.handleRemoveAllPolygons);
+    document.getElementById('doneSelectingArea').addEventListener('click', this.handleDoneSelecting);
+    document.getElementsByName("serviceExpeditionCost").forEach((element) => { 
+      element.addEventListener('click', this.handleExpeditionInfoClick); 
+    });
+    document.getElementById("addBag").addEventListener('click', this.handleAddBagClick);
+    document.getElementById("removeBag").addEventListener('click', this.handleRemoveBagClick);
+    document.getElementById("doneAddOns").addEventListener('click', this.handleDoneAddOns);
+    document.getElementById("quote_email").addEventListener('keyup', () => this.enableSubmitButton);
+    document.getElementById("quote_phone_number").addEventListener('keyup', () => this.enableSubmitButton);
+    document.getElementById('contact-info-form').addEventListener('submit', this.handlePaymentSubmit);
+    document.getElementById('closeSuccess').addEventListener('click', this.reloadPage);
   }
 
-  showAddressModal(){
-    const submitAddressModal = $('#submitAddressModal')
-    submitAddressModal.modal('show');
-    submitAddressModal.keyup(function() {
-      $('input[type=submit]').removeAttr('disabled').removeClass('disabled');
-    });
+  enableFindButton(){
+    document.getElementById('addressSubmit').classList.remove('disabled');
   }
   
   handleSearchSubmit(event) {
     event.preventDefault();
-    $('#submitAddressModal').modal('hide')
+    this.showNextSection('collapseFindAddress', 'collapseMap');
+    document.getElementById('checkMarkAddress').classList.remove('hidden');
+    
     if (this.marker) { this.marker.setMap(null) }
     if (this.polygons.length > 0) {
       this.polygons.forEach(p => { p.setMap(null) });
       this.polygons = [];
-      this.onPolygonsChanged(this.polygons);
+      this.handlePolygonChanged(this.polygons);
+    }
+    let address = document.getElementById('addressInput').value;
+    this.geocodeAddress(address);
+  }
+
+  showNextSection(collapseNode, showNode) {
```

Comment:
Use symmetric names for your variables, either:
- `collapseNode`, `expandNode`, or
- `hideNode`, `showNode` or
- `sectionToHide`, `sectionToShow`